distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -53,138 +53,164 @@ The default metacard type has the following required attributes.
 Though the following attributes are required on all metacard types, setting their values is optional except for ID.
 
 .[[_metacard_required_attributes]]Metacard Required Attributes
-[cols="2m,2m,1,2", options="header"]
+[cols="2m,2m,1m,2,1", options="header"]
 |===
 
 |Metacard Constant
 |Attribute Name
 |Attribute Format
 |Description
+|Constraints
 
-|CONTENT_TYPE
-|metadata-content-type
-|STRING
-|Attribute name accessing for the metadata content type of a resource.
+|CHECKSUM
+|checksum
+|String
+|Checksum value for the primary resource for the metacard
+|< 1024 characters
 
-|CONTENT_TYPE_VERSION
-|metadata-content-type-version
-|STRING
-|Attribute name for the version of the metadata content accessing type of a resource.
+|CHECKSUM_ALGORITHM
+|checksum-algorithm
+|String
+|Algorithm used to calculate the checksum on the primary resource of the metacard.
+|< 1024 characters
 
 |CREATED
 |created
 |DATE
 |Attribute name for the date/time *this metacard* was created.
+|None
```

Comment:
When we use `None` for dates, does it mean they can be empty? Also, don't we have any kind of constraints between dates e.g., expiration date comes after modified date comes after creation date?